packages/web-components/fast-foundation/src/menu-item/stories/menu-item.register.ts

```diff
@@ -244,18 +241,10 @@ const styles = css`
     :host([aria-checked="true"]) .radio-indicator {
         background: var(--foreground-on-accent-rest);
         border-radius: 999px;
-        bottom: 4px;
-        display: block;
-        left: 4px;
-        pointer-events: none;
-        position: absolute;
-        right: 4px;
-        top: 4px;
-    }
-
-    :host([aria-checked="true"]) ::slotted([slot="radio-indicator"]) {
         display: block;
         pointer-events: none;
+        height: 50%;
+        width: 50%;
```

Comment:
Was this broken prior or is this to fix something related to this update?

Reply:
The previous radio indicator styling created a new stacking context (position:absolute) which caused z-order issues.  Since the stacking context seem unnecessary to me it seemed appropriate to remove it.  The old styling will still work until someone encounters a z-order issue...